### DISCUSSION TITLE
Plugin works on my PC, but not on my iPhone

### DISCUSSION THREAD

**Discussion by CH0NZA** [gavvvr/obsidian-imgur-plugin#98] [Getting help]
I noticed that the plugin works fine on my laptop, but not on my iPhone. Whenever I try pasting a picture on my phone, I get an error message saying, "Please configure Client ID for Imgur plugin or disable it". Obsidian will store it locally, which is _okay_, but I installed the plugin to avoid filling up my storage with pictures.

**Comment by gavvvr** [comment]
Hello @CH0NZA 
Did you configure your own ClientId on iPhone? The plugin might be working on PC because it's running the older version, but since [v2.6.0](https://github.com/gavvvr/obsidian-imgur-plugin/releases/tag/2.6.0) it's now [required to generate your own ClientID](https://github.com/gavvvr/obsidian-imgur-plugin?tab=readme-ov-file#generating-client-id), even for anonymous uploads. Each user now generates own ClientID, because sharing single one across all the users leads to hitting rate limits applied by Imgur.

Also, please note that it will work on mobile only on "Paste" action. The plugin does not support adding images from phone gallery yet (#60)
